aws/resource_aws_codebuild_project.go

```diff
@@ -180,6 +180,16 @@ func resourceAwsCodeBuildProject() *schema.Resource {
 				Default:      "60",
 				ValidateFunc: validateAwsCodeBuildTimeout,
 			},
+			"badge_enabled": {
+				Type:     schema.TypeBool,
+				Optional: true,
+				Default:  false,
+			},
+			"badge_url": {
+				Type:     schema.TypeString,
+				Computed: true,
+				Optional: true,
```

Comment:
Since we cannot set the badge URL via the API, `Optional: true` should be removed from this attribute. More information: https://www.terraform.io/docs/extend/schemas/schema-behaviors.html#optional